src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -102,6 +102,8 @@ protected function startSession(Request $request)
 
 		$session->start();
 
+		app('events')->fire('session.started');
```

Comment:
You can't use that helper here. You must inject an implementation of the events contract in the constructor.